Sources/AccountSDKIOSWeb/Simplified Login/View/Fonts/FontManager.swift

```diff
@@ -0,0 +1,29 @@
+//
+// Copyright © 2022 Schibsted.
+// Licensed under the terms of the MIT license. See LICENSE in the project root.
+//
+
+import Foundation
+import CoreGraphics
+import CoreText
+
+struct FontManager {
+
+    public static func registerFonts() {
+        Inter.allCases.forEach {
+            registerFont(bundle: Bundle.accountSDK(for: SimplifiedLoginViewController.self), fontName: $0.rawValue, fontExtension: "ttf")
+        }
+    }
+
+    fileprivate static func registerFont(bundle: Bundle, fontName: String, fontExtension: String) {
+        guard let fontURL = bundle.url(forResource: fontName, withExtension: fontExtension),
+              let fontDataProvider = CGDataProvider(url: fontURL as CFURL),
+              let font = CGFont(fontDataProvider) else {
+            fatalError("Couldn't create font from filename: \(fontName) with extension \(fontExtension)")
```

Comment:
Do we really want to crash the app if we can't parse the font somehow?

Reply:
If app crashes that it means that files in the sdk are somehow corrupted 🤔